feature/src/main/feature/feature.xml

```diff
@@ -516,32 +521,27 @@ limitations under the License.
     </feature>
 
     <feature name='jclouds-cloudsigma2-hnl' description='CloudSigma v2 Honolulu' version='${project.version}'>
-        <feature version='${project.version}'>jclouds-compute</feature>
```

Comment:
Goot point @demobox. I reverted the changes for the sake of consistency. Still think it's the right thing to do though. Perhaps something to take care of in another PR.